### PR TITLE
Update the chapter extensions for Contao 4

### PR DESCRIPTION
**Pull request by lionel-m**

### CODE REVIEW

File: manual/en/04-managing-content/README.md [hunk 1]
```diff
@@ -3,9 +3,12 @@
 The following chapters explain how to manage content in Contao. The Contao core
 supports many different content types like articles, news, events, newsletters
 or forms. Further content types like banners, tickets, products or
-recommendations are available in the [Extension Repository][1]. To create
-content, log into the back end and choose one of the modules in the Content
-section of the navigation menu.
+recommendations are available in the [Extension Repository][1] or in
+[Packagist][2] (See "[Finding extensions or bundles][3]" chapter for more
+detailed information). To create content, log into the back end and choose one
+of the modules in the Content section of the navigation menu.
```
Comment (aschempp) [inline manual/en/04-managing-content/README.md]: Lowercase C for _content_

File: manual/en/05-system-administration/extensions.md [hunk 2]
```diff
@@ -1,58 +1,135 @@
...
+
+An extension can be installed with Composer or manually. With Composer, the
+installation process is the same as a bundle except for the registration in the
+`app/AppKernel.php` file where the code is slightly different.
+
+Add the following line as in the example below by changing the first parameter
+`myExtensionName` with the name of your extension.
+
+```php
+new Contao\CoreBundle\HttpKernel\Bundle\ContaoModuleBundle('myExtensionName', $this->getRootDir()),
```
Comment (aschempp) [inline manual/en/05-system-administration/extensions.md]: I think this line is obsolete if we have an example of `AppKernel.php` right below?

File: manual/en/05-system-administration/extensions.md [hunk 3]
```diff
@@ -1,58 +1,131 @@
...
-Contao will automatically download and install the extension and update the
-database if necessary.
+### Installing a bundle with Composer
 
-![](images/extension-install.jpg)
+A name of a bundle is divided into two parts. The first part is the name of the
+vendor (project owner) and the second the bundle name. For example:
+`contao/news-bundle`.
 
+Run the command `php composer.phar require vendor/bundleName` in your
```
Comment (aschempp) [inline manual/en/05-system-administration/extensions.md]: shouldn't that be `composer require vendor/bundleName` now?

File: manual/en/05-system-administration/extensions.md [hunk 4]
```diff
@@ -1,58 +1,131 @@
...
+    }
+
+    // ...
+}
+```
+
+
+#### Manually
+
+Find the extension you want to install in the [Extension Repository][1] and
```
Comment (aschempp) [inline manual/en/05-system-administration/extensions.md:106]: I would add an intro like this:

> Almost all extensions for Contao 3 are also compatible with Contao 4. If a Contao 3 extension is not available on Composer, you can install it manually and try if it works with Contao 4. Be aware that you must also take care of installing all dependencies as listed in the Extension Repository.

Comment (lionel-m) [reply manual/en/05-system-administration/extensions.md:106]: Added in 1c835f69b673ed6650538264ef7dc4e37db19b4a

File: manual/en/05-system-administration/extensions.md (hunk 3 above)
Comment (lionel-m) [reply manual/en/05-system-administration/extensions.md]: Modified in 1c835f69b673ed6650538264ef7dc4e37db19b4a

**Comment by aschempp** [comment]
Looks RTM to me 👍 
Want to add the french or a separate pull request?

**Comment by lionel-m** [comment]
I'll add the translation in this PR. :smiley:

File: manual/en/05-system-administration/extensions.md [hunk 5]
```diff
@@ -132,5 +132,5 @@ release.
 [3]: https://getcomposer.org/doc/00-intro.md#introduction
 [4]: https://packagist.org/search/?q=&type=contao-bundle
 [5]: https://packagist.org/search/?q=&type=contao-module
-[6]: ../01-installation/installing-contao.md#symbolic-link
+[6]: https://docs.contao.org/books/cookbook/en/folder-structure.html#symbolic-link
```
Comment (lionel-m) [inline manual/en/05-system-administration/extensions.md]: @aschempp Initially, the link targeted the chapter "Installing Contao" but then we transferred this paragraph into the cookbook. Are you agree to add a cookbook chapter as a reference? Manual -> Cookbook

**Comment by lionel-m** [comment]
It's RTM for me too if you agree with https://github.com/contao/docs/pull/425/commits/7d559313d69c82581e2005ff4b12d028b081d397#r83456222

File: manual/en/05-system-administration/extensions.md (hunk 5 above)
Comment (aschempp) [reply manual/en/05-system-administration/extensions.md]: In general I'm fine with that, but in this case maybe we can just link to the Wikipedia article like the cookbook does? The sentence looks like an explanation what _symbolic link_ is, and not about what Contao does with it (was described in the cookbook).

Comment (lionel-m) [reply manual/en/05-system-administration/extensions.md]: I added the Wikipedia links 👍

Comment (lionel-m) [reply manual/en/05-system-administration/extensions.md]: Modified in 06a85b681356e3f4331655bea29f23bce65eb91f

**Comment by lionel-m** [comment]
@aschempp: It is complete (RTM)

**Comment by aschempp** [comment]
Would someone be available for the german translation? @fritzmg maybe? 😊

**Comment by fritzmg** [comment]
Sure, I'll have a look at it :)

File: manual/en/04-managing-content/README.md [hunk 6]
```diff
@@ -3,9 +3,12 @@
 The following chapters explain how to manage content in Contao. The Contao core
 supports many different content types like articles, news, events, newsletters
 or forms. Further content types like banners, tickets, products or
-recommendations are available in the [Extension Repository][1]. To create
-content, log into the back end and choose one of the modules in the Content
-section of the navigation menu.
+recommendations are available in the [Extension Repository][1] or in
+[Packagist][2] (See the chapter "[Finding extensions or bundles][3]" for more
```
Comment (fritzmg) [inline manual/en/04-managing-content/README.md]: Hm, this is only for Contao 4, right? Then the mention of the ER should be removed completely, I think?

Comment (lionel-m) [reply manual/en/04-managing-content/README.md]: We can write:
>like articles, news, events, newsletters or forms. Further content types like banners, tickets, products or recommendations are available as bundles (See the chapter...

File: manual/en/05-system-administration/extensions.md [hunk 7]
```diff
@@ -1,58 +1,136 @@
...
-functionality. There are more than 1,400 extensions available in the Contao
-Extension Repository, which you can browse directly in the back end.
-Communication with the repository server is done via SOAP, so you need to enable
-the PHP SOAP extension to use the service (if not enabled by default).
+functionality. There are more than 1,800 extensions available in the Contao
+[Extension Repository][1].
 
+Contao 4 is built on top of the Symfony framework and takes advantage of its
+functionalities but also of its terminology. In a Symfony project, an extension
+is named a bundle.
```
Comment (fritzmg) [inline manual/en/05-system-administration/extensions.md]: I would rephrase this whole paragraph to:

> Extensions are an essential part of Contao, because they allow you to add extra functionality. Contao 4 is based on the Symfony framework and takes advantage of its functionalities and its terminiology. In Contao 4 you can use classic extensions from previous Contao versions as well as so called _Bundles_.

**Comment by lionel-m** [comment]
@aschempp, @fritzmg This PR is ready to be translated into German.

**Comment by aschempp** [comment]
@fritzmg when could you take a look at this? 😊

**Comment by fritzmg** [comment]
Yes, sorry, I already started working on it but couldn't finish it yet. It's a little more effort than the other changes.
